.github/workflows/main.yml

```diff
@@ -5,8 +5,10 @@ on:
     branches:
       - master
     tags:
-      - '*'
```

Comment:
Do you want/need to keep the `tags` key?

Reply:
Not sure about that. If we always use the releases to deploy new version, tags are not needed. Besides, the last step would contain 
```
if: github.event_name == 'release'
```
instead of 
```
if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags/')
```

Reply:
no, we might release with a manual tag if something broke

Reply:
I'd say yes, we always use releases, as our workflow is to create a release from a draft release on github.

Reply:
That being said, the "tags" argument didn't work :( so, not sure about that. I don't know.